readme.md

```diff
@@ -96,6 +96,13 @@ const deletedPaths = await deleteAsync(['temp/*.js'], {dryRun: true});
 console.log('Files and directories that would be deleted:\n', deletedPaths.join('\n'));
 ```
 
+##### dot
+
+Type: `boolean`\
+Default: `false`
+
+Causes `*` to treat files beginning with a `.` character like any other file. If `false`, dot files will not be matched by `*`. Alternately, `.*` can be used to match dot files. This option is passed through to [`glob`](https://github.com/isaacs/node-glob#dots).
```

Comment:
It's passed to https://github.com/mrmlnc/fast-glob#dot

I also think this could be written more succinctly.

> Allow patterns to match files/folders that begin with a period (.).

And maybe also a short example, like in the fast-glob docs.